skimage/measure/tests/test_regionprops.py

```diff
@@ -1420,7 +1427,9 @@ def test_extra_properties():
 
 def test_extra_properties_intensity():
     region = regionprops(
-        SAMPLE, intensity_image=INTENSITY_SAMPLE, extra_properties=(intensity_median,)
+        SAMPLE,
+        intensity_image=INTENSITY_SAMPLE,
+        extra_properties=(custom_intensity_median,),
     )[0]
     assert region.intensity_median == np.median(INTENSITY_SAMPLE[SAMPLE == 1])
```

Comment:
I think this is not testing what it's supposed to. The new "extra property" is now named `custom_intensity_median`. And since the name of the new property is derived from the function you need to test for that attribute

```suggestion
    assert region.custom_intensity_median == np.median(INTENSITY_SAMPLE[SAMPLE == 1])
```

this applies for other changes as well...

Reply:
Ah, yes, that was poor refactoring, I'll update the PR.

Reply:
I now split the PR into a first commit for the refactoring of the `extra_properties` tests to use `custom_intensity_median` (to ensure it is passing the tests), and a second commit adding the new `intensity_median` feature.